lib/galaxy/dependencies/__init__.py

```diff
@@ -69,7 +69,8 @@ def load_job_config_dict(job_conf_dict):
                     pass
             else:
                 try:
-                    job_conf_dict = yaml.safe_load(job_conf_path)
+                    with open("job_conf_path", "r") as f:
```

Comment:
Maybe we can also tighten the
`if '.xml' in job_conf_path:` with `if job_conf_path.endswith('.xml')` ?

Reply:
```.xml.sample`` though